database/mongo/seed.js

```diff
@@ -0,0 +1,65 @@
+const faker = require('faker/locale/en_US');
+const Jabber = require('jabber');
+const moment = require('moment');
+const Review = require('./index.js');
+// const Review = require('./mongoSchema');
+
+// moment(randomDate).fromNow()
+
+const buzzWords = ['great food', 'eat', 'service', 'drinks', 'loved', 'never again', 'dessert', 'main menu', 'appetizers', 'enjoyed', 'hated', 'server', 'noise', 'waitress', 'waiter', 'slow', 'fast', 'go again', 'clean', 'dirty', 'kids', 'our group', 'slow service', 'cold food', 'awesome drinks', 'fun time', 'romantic atmosphere', 'dinner', 'lunch', 'breakfast', 'hot plate', 'smelled wonderful', 'no alcohol', 'bartender', 'wine list', 'outdoor seating', 'large room', 'no tips', 'valet parking', 'fine dining', 'dishes', 'shrimp', 'steak', 'chicken', 'beef', 'pasta', 'cake', 'pastries', 'creme brulee', 'simmering', 'coffee', 'fresh food', 'candlelit', 'anniversary', 'birthday', 'party', 'with friends', 'evening', 'stuffed', 'would go again', 'recommend this place', 'restrooms clean', 'host greeting', 'lovely place', 'best view', 'indoor seating', 'waited an hour', 'risotto', 'baked salmon', 'fried rice', 'shrimp scampi', 'sushi', 'wait time'];
+
+const tags = ['Good for groups', 'Desserts', 'Appetizers', 'Drinks', 'Kid friendly'];
+
+const circleColors = ['#6c8ae4', '#d86441', '#bb6acd', '#df4e96'];
+
+const jabber = new Jabber(buzzWords, 2);
+
+const createReview = (resId) => {
+  const review = {};
+  review.restaurantId = resId;
+  review.firstName = (Math.random() < 0.7) ? faker.name.firstName() : '';
+  if (review.firstName.length) {
+    review.lastName = (Math.random() < 0.7) ? faker.name.lastName() : '';
+  } else {
+    review.lastName = '';
+  }
```

Comment:
If I'm following, you're basically determining whether the reviewer has a name? I'm not quite certain the benefit of that, but if that's the goal this looks great.

Reply:
I was confused as well.  After looking through it, you can review 'anonymously' and if the first and last names are blank, the front end makes the name show up as 'OpenTable Diner'.  Really interesting way to go about it.